CMakeLists.txt

```diff
@@ -43,6 +43,12 @@ option(IREE_BUILD_DEBUGGER "Builds the IREE debugger app." OFF)
 option(IREE_BUILD_PYTHON_BINDINGS "Builds the IREE python bindings" OFF)
 option(IREE_BUILD_EXPERIMENTAL "Builds experimental projects." OFF)
 
+#TODO: Make is functional!
+set(IREE_HAL_BACKENDS_TO_BUILD "all"
```

Comment:
We usually call these "drivers" for the runtime part. I think we should be consistent here

Reply:
This was a suggestion by @benvanik. But, yes, I will change it and if it needs another change, you might like to edit it during integration :)